core/controllers/pages_test.py

```diff
@@ -15,14 +15,149 @@
 """Tests for various static pages (like the About page)."""
 
 from core.tests import test_utils
+import feconf
 
 
-class NoninteractivePagesTests(test_utils.GenericTestBase):
+class AboutPageTest(test_utils.GenericTestBase):
+    """Test for About page."""
 
     def test_about_page(self):
-        """Test the About page."""
-        response = self.get_html_response('/about')
+        response = self.testapp.get('/about')
+        self.assertEqual(response.status_int, 200)
         self.assertEqual(response.content_type, 'text/html')
         response.mustcontain(
-            'I18N_ABOUT_PAGE_CREDITS_TAB_HEADING',
-            'I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_5_LICENSE_HEADING')
+            'I18N_ABOUT_PAGE_CREDITS_TAB_HEADING')
+
+
+class SplashPageTest(test_utils.GenericTestBase):
+    """Test for splash page."""
+
+    def test_splash_page(self):
+        response = self.testapp.get('/splash')
+        self.assertEqual(response.status_int, 200)
+        self.assertEqual(response.content_type, 'text/html')
+        response.mustcontain('I18N_SPLASH_TITLE')
+
+
+    def test_splash_page_with_c_value(self):
+        """Test for splash page when c value is included."""
+        response = self.testapp.get('/splash?c=c')
+        self.assertEqual(response.status_int, 302)
+        self.assertEqual(response.content_type, 'text/html')
+
+
+class GetStartedPageTest(test_utils.GenericTestBase):
+    """Test for get started page."""
+
+    def test_get_started_page(self):
```

Comment:
Can you please change the method name to follow this pattern:

`test_action_withCondition_withAnotherCondition_hasExpectedOutcome`

In this case, maybe we can write "test_loading_get_started_page_loads_correctly" or something around this? (Ditto everywhere.)

(See [guidelines on writing backend](https://github.com/oppia/oppia/wiki/Writing-backend-tests#some-general-guidelines-for-writing-good-tests) test for more info.)

Reply:
Done!

Reply:
Hello @DubeySandeep, would review my request.